timemachine/potentials/nonbonded.py

```diff
@@ -300,6 +305,13 @@ def apply_cutoff(x):
     qij = apply_cutoff(charges[inds_l] * charges[inds_r])
     electrostatics = direct_space_pme(dij, qij, beta)
 
+    if rescale_mask is not None:
+        assert len(rescale_mask) == len(pairs)
```

Comment:
nit: strengthen shape assertion
```suggestion
        assert rescale_mask.shape == (len(pairs), 2)
```